src/EA.Iws.RequestHandlers.Tests.Unit/NotificationMovements/BulkUpload/PrenotificationExcessiveShipmentsRuleTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace EA.Iws.RequestHandlers.Tests.Unit.NotificationMovements.BulkUpload
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Core.Movement.Bulk;
+    using Core.Rules;
+    using Domain.Movement;
+    using FakeItEasy;
+    using Prsd.Core.Mediator;
+    using RequestHandlers.NotificationMovements.BulkUpload;
+    using Xunit;
+
+    public class PrenotificationExcessiveShipmentsRuleTests
+    {
+        private readonly IMediator mediator;
```

Comment:
I can't see the mediator being used within this test so perhaps this can be removed. Noticed it on other tests too but just leaving this comment here rather than duplicating them :)